packages/babel-plugin-proposal-json-strings/src/index.js

```diff
@@ -0,0 +1,23 @@
+import { declare } from "@babel/helper-plugin-utils";
+import syntaxJsonStrings from "@babel/plugin-syntax-json-strings";
+
+export default declare(api => {
+  api.assertVersion(7);
+  const regex = /(^|[^\\])([\u2028\u2029])/g;
```

Comment:
This type of check with `[^\\]` is always a bit dangerous because it will fail on code like

    "before\\ after";

since there _is_ a slash beforehand, but it was already escaped and thus not an issue. Not super common, but it is an edge case that isn't handled here.

Reply:
Good catch, fixed.